react/src/Footer.js

```diff
@@ -2,15 +2,18 @@ import { useContext } from 'react';
 import HeaderData from './HeaderData';
 import Env from './utils/Env';
 import Image from './utils/Image';
+import { useFetching } from './utils/Fetch';
 
 const Footer = (props) => {
 
     const [ header ] = useContext(HeaderData);
+    const [ fetching ] = useFetching();
 
     // TODO
     // Get the URLs CGAP/4DN URLs below directly from the Portal.
 
     if (header.loading) return null;
+    if (fetching.length > 0) return null;
```

Comment:
Taking this out; it was an experiment I left in by accident. The intention was to try to get rid of the flashing of the (Harvard branding) foot on the page at the top of the page before other page content was loaded. So this was seeing if there were any outstanding fetches and if so it would not display the footer at all, unless/until all outstanding fetches were complete. I don't think this is a good/consistent way of doing this. Not a big deal UI wise.

Reply:
This was slightly tricker to get right WRT centering, because of the left and right content. I'm sure I'm probably doing it wrong CSS-wise. I added TODO comment to clean this up.